src/web/screens/addTeamMemberDialog.scss

```diff
@@ -1,3 +1,9 @@
 .addTeamMemberDialog {
   max-width: 510px;
 }
+
+@media (min-width: 606px) {
```

Comment:
See my PM - this makes the dialog too narrow once the window is less than 606px, we ideally want to make it full-width for any smaller window, and stop it from growing once it reaches the target width.